feature-utils/poly-look/src/react-components/filterChips/filterChips.jsx

```diff
@@ -120,17 +123,34 @@ const ChipGroup = ({
       {extendedChipsContent.map((e) => {
         const id = chipId(e);
         return (
-          <button
-            className={isChipActive(id) ? "chip selected" : "chip"}
-            onClick={() => handleChipClick(id)}
-            key={id}
-          >
-            {e.translation || id}
-          </button>
+          <div className="chip-container" key={id}>
+            <Chip
+              id={id}
+              translation={e.translation}
+              handleClick={handleChipClick}
+              active={isChipActive(id)}
+            />
+          </div>
         );
       })}
     </div>
   );
 };
 
-export default ChipGroup;
+FilterChips.propTypes = {
+  chipsContent: propTypes.oneOfType([propTypes.string, propTypes.object])
+    .isRequired,
+  defaultActiveChips: propTypes.array,
+  onChipClick: propTypes.func,
+  exclusive: propTypes.bool,
+  theme: propTypes.string,
+  allChip: propTypes.object,
+  othersChip: propTypes.object,
```

Comment:
Please review these for requiredness. I'd say that at least `exclusive` would need to have a definite value.